src/status_im/ui/screens/offline_messaging_settings/views.cljs

```diff
@@ -52,7 +52,7 @@
                   {:keys [use-mailservers?]} [:multiaccount]]
     [react/view {:style styles/wrapper}
      [topbar/topbar
-      {:title (i18n/label :t/history-nodes)
+      {:title (i18n/label :t/status-nodes)
```

Comment:
This will probably break things. In the translation file, this is called "history-nodes" (line 536 of `en.json`), but here you are referring to "status-nodes". For this to work, there should be a key value pair called `"status-nodes": "Status nodes"` in `en.json`. 

Any`key:value` in `en.json` can be accessed as `:t/key`.

I suggest that you undo all cahnges to cljs files and let the changes in `en.json` stay. That will achieve the required result.